shell/browser/electron_browser_client.cc

```diff
@@ -785,7 +786,11 @@ ElectronBrowserClient::CreateClientCertStore(
     content::BrowserContext* browser_context) {
 #if BUILDFLAG(USE_NSS_CERTS)
   return std::make_unique<net::ClientCertStoreNSS>(
-      net::ClientCertStoreNSS::PasswordDelegateFactory());
+      base::BindRepeating([](const net::HostPortPair& server) {
+        crypto::CryptoModuleBlockingPasswordDelegate* delegate =
+            new ChromeNSSCryptoModuleDelegate(server);
+        return delegate;
+      }));
```

Comment:
Are we losing functionality by replacing the `net::ClientCertStoreNSS` instance? I'm seeing some logic in [client_cert_store_nss.cc](https://source.chromium.org/chromium/chromium/src/+/main:net/ssl/client_cert_store_nss.cc) which seems potentially necessary.

Reply:
I'm not sure what the concern is. The patch does not replace ClientCertStoreNSS. It just provides a PasswordDelegateFactory.

Reply:
I misread this, my bad. The code as you wrote it looks good 👍